codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/customizations/ResiliencyConfigCustomization.kt

```diff
@@ -221,13 +225,14 @@ class ResiliencyReExportCustomization(private val runtimeConfig: RuntimeConfig)
             writer.rustTemplate(
                 """
                 pub use #{sleep}::{AsyncSleep, Sleep};
-                pub use #{types_retry}::{RetryConfig, RetryConfigBuilder};
-                /// Config structs required by [`RetryConfig`]
+                /// Retry configuration
                 pub mod retry {
-                    pub use #{types_retry}::RetryMode;
-                    pub use #{types_retry}::RetryKind;
+                    pub use #{types_retry}::{RetryConfig, RetryConfigBuilder, RetryMode};
+                }
+                /// Timeout configuration
```

Comment:
Should we add a note about these being reëxports so people understand that configs from separate crates are compatible?